### PR TITLE
fix(topology): remove usage of pf react icons

### PR DESCRIPTION
**Pull request by debsmita1**
**Resolves:**
https://issues.redhat.com/browse/RHIDP-2600

**Screenshot:**
https://docs.google.com/document/d/1sreb-7_MiFQ_Ei4GV1ZZV6C6vX2Nd0i2k_ch8enDbok/edit

### CODE REVIEW

**Comment by ShiranHi** [comment]
@debsmita1 looks good, thank you!

**Comment by debsmita1** [comment]
@ShiranHi I just noticed that the topology icon that I have used here, the same has been used for the navigation icon for clusters on the navigation sidebar in your designs https://www.figma.com/design/6ORS1mhUaxeyTBjMmE95Rn/Home-page-redesign?node-id=164-98&t=QOSyQcsAQcdDUKxB-0

**Comment by ShiranHi** [comment]
> @ShiranHi I just noticed that the topology icon that I have used here, the same has been used for the navigation icon for clusters on the navigation sidebar in your designs https://www.figma.com/design/6ORS1mhUaxeyTBjMmE95Rn/Home-page-redesign?node-id=164-98&t=QOSyQcsAQcdDUKxB-0

We may change the icon in the side navigation for this plugin (you can see WIP and discussion about it [here](https://docs.google.com/presentation/d/1PpStlGtPReaYBgFJkLpyintWaXhzFFQbScLA9l-hGm0/edit#slide=id.g2e26b795725_0_12)).
For now, I think this is ok...

File: plugins/topology/src/components/Graph/decorators/PipelineRunDecorator.tsx [hunk 1]
```diff
@@ -6,9 +6,9 @@ import {
   getLatestPipelineRun,
   getTaskRunsForPipelineRun,
   pipelineRunStatus,
+  Status,
```
Comment (karthikjeeyar) [inline plugins/topology/src/components/Graph/decorators/PipelineRunDecorator.tsx:9]: In PipelineRunDecorator, success icon (check mark) should be green in colour.

image

Comment (karthikjeeyar) [reply plugins/topology/src/components/Graph/decorators/PipelineRunDecorator.tsx:9]: Looks like there isn't any straight forward way to override PF styles here, I did wrap the Status Icon component with  overriding SVG fill property to retain the theme colors. See if this approach works for you https://gist.github.com/karthikjeeyar/c584caa00d3da30b06ab41269b726a01

Comment (debsmita1) [reply plugins/topology/src/components/Graph/decorators/PipelineRunDecorator.tsx:9]: Fixed. Thank you for the suggestion
Screenshot 2024-09-18 at 2 33 56 PM

File: plugins/tekton/src/components/PipelineRunList/PipelineRunVulnerabilities.tsx [hunk 2]
```diff
@@ -87,8 +92,8 @@ const PipelineRunVulnerabilities: React.FC<PipelineRunVulnerabilitiesProps> = ({
             <span className={classes.severityStatus}>
               <Tooltip content="High">
                 <AngleDoubleUpIcon
-                  title="High"
                   className={classes.highStatus}
+                  component={high}
```
[the remark this answers is not in the document]
Comment (debsmita1) [reply plugins/tekton/src/components/PipelineRunList/PipelineRunVulnerabilities.tsx]: Fixed!
Screenshot 2024-09-18 at 2 35 12 PM

**Review by jerolimov — APPROVED** [review]
/lgtm

**Pull request merged**